### ISSUE TITLE
Importing graph data in Windows

### ISSUE THREAD

**Issue by ojpbay** [triAGENS/ArangoDB-Data#4]
I have recently been experimenting with ArangoDB for Windows. I am able to import some of the sample data here using arangoimp.exe. I would like to be able to try some sample queries with the graphs you have in this repository, e.g. IMDB. I cannot see how I can import this on the Windows platform. Can anyone help? Many thanks.

**Comment by dothebart** [comment]
Try running arangosh, and then issue 
arangosh [_system]> require("internal").load("c:\\...wherever\\ArangoDB-Data\\Graphs\\IMDB/import.js")
which should import properly

**Comment by ojpbay** [comment]
I did try that. The path is correct but I get the following


arangosh [_system]> require("internal").load("c:\Temp\IMDB\import.js")
JavaScript exception: [ArangoError 2: cannot read file: system error]
!require("internal").load("c:\Temp\IMDB\import.js")
!                    ^
stacktrace: Error: cannot read file: system error
    at Error (native)
    at <shell command>:1:21
arangosh [_system]> require("internal").load("c:\Temp\IMDB\import.js")
JavaScript exception: [ArangoError 2: cannot read file: system error]
!require("internal").load("c:\Temp\IMDB\import.js")
!                    ^
stacktrace: Error: cannot read file: system error
    at Error (native)
    at <shell command>:1:21

**Comment by fceller** [comment]
@ojpbay is it possible that you need to escape \ in the shell? Can you try

require("internal").load("c:\\Temp\\IMDB\import.js")

or

require("internal").load("c:/Temp/IMDB\import.js")

**Comment by ojpbay** [comment]
Thanks but unfortunately still not working. I noticed in the first reply the different forward / backslashes. Trying your combinations results int he same error. I've tried running the shell with elevated permissions...

One thing I did notice, using the following (all forward slashes) I get a different error message - the directory does exist btw.

arangosh [_system]> require("internal").load("c:/Temp/IMDB/import.js")
JavaScript exception in file 'c:/Temp/IMDB/import.js' at 62,12: [ArangoError 2: cannot open file: No such file or directory]
!  internal.processJsonFile("nodes.json", storeVertex);
!           ^
stacktrace: Error: cannot open file: No such file or directory
    at Error (native)
    at c:/Temp/IMDB/import.js:62:12
    at c:/Temp/IMDB/import.js:64:2
    at <shell command>:1:21
